.github/CODEOWNERS

```diff
@@ -1,14 +1,16 @@
+# GENERATED by @kbn/dev-utils (Do not edit manually)
```

Comment:
I took this file from master, removed all the `#` and empty lines, and then sorted all of the other lines. Did the same with this version and got this diff: https://gist.github.com/spalger/b849067c9c6b62d70c11238102e2e201

I suggest doing something similar to understand the changes made here a little more clearly.

Some things I'm noticing:
 - seems the pulse team was renamed to `@elastic/kibana-telemetry`
 - Seems there are a number of codeowner changes related to design teams which are not reflected here, including teams like `@elastic/kibana-core-ui-designers`, `@elastic/observability-design`, and `@elastic/security-design`

Reply:
Ok, I was trying to go off of teams that do indeed exist.  I validated the names here: https://github.com/orgs/elastic/teams?type=source

To point, `pulse` is not defined.

Reply:
This PR [continues to reference `pulse`](https://github.com/elastic/kibana/blob/8a314b67e2b35fd06f22256bbaeb528c8bd86a86/.github/CODEOWNERS#L231-L238)... I'm confused... I'm saying that it seems the `pulse` team was renamed to `telemetry`